demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -1,210 +1,52 @@
-
-        content_item_id = _get_file_id(self.structure_validator.scheme_name, self.current_file)
-        file_type = self.structure_validator.scheme_name
-        # Test playbook case
+        formatted_error = f"{file_path}: [{error_code}] - {error_massage}".rstrip("\n") + "\n"
 
-        if 'TestPlaybooks' in self.file_path and file_type == 'playbook':
-            is_configured_test = any(test_config for test_config in conf_json_tests if
-                                     is_test_config_match(test_config, test_playbook_id=content_item_id))
-            if not is_configured_test:
-                missing_test_playbook_configurations = json.dumps({'playbookID': content_item_id}, indent=4)
-                missing_integration_configurations = json.dumps(
-                    {'integrations': '<integration ID>', 'playbookID': content_item_id},
-                    indent=4)
-                error_message = \
-                    f'The TestPlaybook {content_item_id} is not registered in {self.CONF_PATH} file.\n' \
-                    f'Please add\n{missing_test_playbook_configurations}\n' \
-                    f'or if this test playbook is for an integration\n{missing_integration_configurations}\n' \
-                    f'to {self.CONF_PATH} path under \'tests\' key.'
-                print_error(error_message)
-                return False
+        if self.should_ignore_error(error_code):
+            if should_print and self.print_as_warnings:
+                click.secho(formatted_error, fg="yellow")
+            return None
 
-        # Integration case
-        elif file_type == 'integration':
-            is_configured_test = any(
-                test_config for test_config in conf_json_tests if is_test_config_match(test_config,
-                                                                                       integration_id=content_item_id))
-            if not is_configured_test:
-                missing_test_playbook_configurations = json.dumps(
-                    {'integrations': content_item_id, 'playbookID': '<TestPlaybook ID>'},
-                    indent=4)
-                no_tests_key = yaml.dump({'tests': ['No tests']})
-                error_message = \
-                    f'The following integration is not registered in {self.CONF_PATH} file.\n' \
-                    f'Please add\n{missing_test_playbook_configurations}\nto {self.CONF_PATH} ' \
-                    f'path under \'tests\' key.\n' \
-                    f'If you don\'t want to add a test playbook for this integration, ' \
-                    f'please add \n{no_tests_key}to the ' \
-                    f'file {self.file_path} or run \'demisto-sdk format -p {self.file_path}\''
-                print_error(error_message)
-                return False
-        return True
+        if should_print:
+            if suggested_fix:
+                click.secho(formatted_error[:-1], fg="bright_red")
```

Comment:
why the -1?

Reply:
in case of a suggested fix, I dont want it to drop a line here - but after the suggestion - it looks better